ui/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-azureextensionui",
     "author": "Microsoft Corporation",
-    "version": "0.33.6",
+    "version": "0.33.7",
```

Comment:
You forgot to bump package-lock.json, but I already bumped it to "0.33.7" in a PR I just merged, so it doesn't really matter. We can include everything together